### PR TITLE
fix: Update privacy policy to include Fingerprint usage [SW-357]

### PR DESCRIPTION
**Pull request by usame-algan**
## What it solves

Resolves [SW-357](https://www.notion.so/safe-global/Update-privacy-policy-1228180fe57380ae9e5dd04da9e084f9)

## How this PR fixes it

- Adds a section for Fingerprint to the privacy policy

## How to test it

1. Open the privacy policy page
2. Observe a new section regarding Fingerprint

## Checklist
* [ ] I've tested the branch on mobile 📱
* [ ] I've documented how it affects the analytics (if at all) 📊
* [ ] I've written a unit/e2e test for it (if applicable) 🧑‍💻

### CODE REVIEW

File: src/markdown/privacy/privacy.md [hunk 1]
```diff
@@ -562,7 +585,7 @@ If you wish to raise a complaint on how we have handled your personal data, you
 
 #### Right to lodge a complaint with a relevant supervisory authority {#right-to-lodge-a-complaint-with-a-relevant-supervisory-authority}
 
-We encourage you to contact us at privacy@cc0de.dev if you have any privacy related concerns. Should you disapprove of the response we have provided you, you have the right to lodge a complaint with our supervisory authority, or with the data protection authority of the European member state you live or work in. The details of the supervisory authority responsible for Berlin, Germany, are:
```
Comment (usame-algan) [inline src/markdown/privacy/privacy.md:565]: This seems to have been added by accident at some point. `cc0x.dev` is the correct one.

**Review by katspaugh — APPROVED** [review]
Approved but as discussed you should rebase this to the `release` branch.

**Comment by francovenica** [comment]
Not a big deal, but there is a empty line between the listed numbers in 3.5.8 . I don't see it that way in the code or anywhere else in the doc, so I assume is not formatted properly

image

**Comment by francovenica** [comment]
LGTM

**Pull request merged**